hypothesis-python/tests/conjecture/test_engine.py

```diff
@@ -1527,6 +1527,19 @@ def test(data):
         assert d2.status == Status.VALID
 
 
+def test_draw_bits_partly_from_prefix_and_partly_random():
+    # a draw_bits call which straddles the end of our prefix has a slightly
+    # different code branch.
+    def test(data):
+        # float consumes draw_bits(64)
+        data.draw_float()
+
+    with deterministic_PRNG():
+        runner = ConjectureRunner(test, settings=TEST_SETTINGS)
+        d = runner.cached_test_function(bytes(10), extend=100)
+        assert d.status == Status.VALID
```

Comment:
drawing one float uses <10 bytes, I think?

Reply:
It's more - a bunch for the sampler, 1 for the sign, and 8 for the value. I only added this test because the branch mentioned in the comments was uncovered, so I'm pretty confident this test does indeed execute that branch.